Dockerfile

```diff
@@ -2,9 +2,9 @@ FROM scratch
 MAINTAINER Maciej Sieczka
 
 # Set the $architecture ARG on your `docker build' command line with `--build-arg architecture=x86_64' or `i686'.
-ARG architecture
+ARG architecture=x86_64
 
-ADD bootstrap.tar.gz /
+ADD bootstrap-${architecture}.tar.gz /
```

Comment:
I wan't to upload and `ADD` `bootstrap.tar.gz` only, ever, to keep `docker build` process simple and fast.